troncos/traces/__init__.py

```diff
@@ -1,26 +1,47 @@
 import logging
 import os
-from typing import Iterable, List
+import sys
+from typing import IO, Iterable, List
 
 import opentelemetry.trace
+
+try:
+    from opentelemetry.exporter.otlp.proto.grpc import (  # type: ignore
+        trace_exporter as grpc_trace_exporter,
+    )
+except ImportError:  # pragma: no cover
+    grpc_trace_exporter = None
+
 from opentelemetry.exporter.otlp.proto.http import trace_exporter
 from opentelemetry.sdk.resources import Attributes, Resource
 from opentelemetry.sdk.trace import SpanProcessor, TracerProvider
 from opentelemetry.sdk.trace.export import (
     BatchSpanProcessor,
     ConsoleSpanExporter,
     SimpleSpanProcessor,
+    SpanExporter,
 )
 
 # noinspection PyProtectedMember
 from opentelemetry.util._once import Once
 
-from troncos.traces.dd_exporter import OTLPSpanExporterDD
+from troncos.traces.dd_exporter import OTLPGrpcSpanExporterDD, OTLPHttpSpanExporterDD
 
 _GLOBAL_SPAN_PROCESSORS: list[SpanProcessor] | None = None
 _GLOBAL_SPAN_PROCESSORS_SET_ONCE = Once()
 
-_DEBUG_SPAN_PROCESSOR: SpanProcessor = SimpleSpanProcessor(ConsoleSpanExporter())
+if out_file := os.environ.get("TRACING_DEBUGGING_FILE", None):
+    traces_out: IO[str] = open("traces.json", "w")
+else:
+    traces_out = sys.stdout
+_DEBUG_SPAN_PROCESSOR: SpanProcessor = SimpleSpanProcessor(
+    ConsoleSpanExporter(out=traces_out)
+)
```

Comment:
trace volume can be pretty intense. Much nicer to shove it in a file than in the terminal.